spec/features/adjustment_spec.rb

```diff
@@ -64,4 +64,17 @@
 
     expect(page).to have_css("table tr", count: 2)
   end
+
+  scenario "User can filter the #index by user" do
+    storage_location = create(:storage_location, name: "here", organization: @organization)
+    storage_location2 = create(:storage_location, name: "there", organization: @organization)
+    create(:adjustment, organization: @organization, storage_location: storage_location, user_id: @user.id)
+    create(:adjustment, organization: @organization, storage_location: storage_location2, user_id: @organization_admin.id)
+
+    visit url_prefix + "/adjustments"
+    select @user.name, from: "filters_by_user"
+    click_button "Filter"
+
+    expect(page).to have_css("table tr", count: 2)
```

Comment:
Not a blocker:  I `think` if you specified `table tbody tr`, you could use `count: 1`, which more clearly communicates the intent of the spec. 

No need to change though; I'm certain we have other specs that do it exactly like this. :)